### PR TITLE
Use new TLS algorithm defaults in remoting tests

### PR DESCRIPTION
**Pull request by ignasi35**
References #29073

### CODE REVIEW

**Comment by patriknw** [comment]
so this is like a follow up on https://github.com/akka/akka/pull/29049 ?

File: akka-remote/src/test/scala/akka/remote/artery/tcp/TlsTcpSpec.scala [hunk 1]
```diff
@@ -33,15 +33,15 @@ class TlsTcpWithSHA1PRNGSpec
...
+      enabled-algorithms = ["TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"]
     }
     """))
 
 class TlsTcpWithDefaultRNGSecureSpec
     extends TlsTcpSpec(ConfigFactory.parseString("""
     akka.remote.artery.ssl.config-ssl-engine {
       random-number-generator = ""
-      enabled-algorithms = ["TLS_RSA_WITH_AES_128_CBC_SHA"]
+      enabled-algorithms = ["TLS_DHE_RSA_WITH_AES_256_GCM_SHA384"]
```
Comment (ignasi35) [inline akka-remote/src/test/scala/akka/remote/artery/tcp/TlsTcpSpec.scala:44]: In Artery tests I'm a little more aggressive at dropping the old algorithm.

**Comment by ignasi35** [comment]
> so this is like a follow up on #29049 ?

Yeah, I added some more background on the issue #29073 :-)

In #29049 I was too quick at editing and pushing code and didn't realise other areas might not be reading the config but duplicating it.

**Review by patriknw — APPROVED** [review]
LGTM

**Review by raboof — APPROVED** [review]

**Comment by raboof** [comment]
manually checked travis is green https://travis-ci.org/github/akka/akka/builds/687524312

**Pull request merged**